test/parsing/errors/labelIneligibleStmt-detailed.good

```diff
@@ -0,0 +1,8 @@
+─── syntax in labelIneligibleStmt.chpl:1 [LabelIneligibleStmt] ───
+  Cannot label 'FnCall' statement.
```

Comment:
Still not super sure about putting `FnCall` (or other AST node types) into user-facing error messages. This would be a first.

Reply:
Unfortunately, I don't think there is. I'm in favor of removing the node name.